HIPs/hedera-hashgraph.md

```diff
@@ -0,0 +1,304 @@
+Once these final steps are completed, we believe the project will be ready for Graduated status and will apply for
+this designation immediately.
+We anticipate that this designation process will proceed swiftly if the TOC concurs that all criteria have been met.
+
+# Solution
+A `PROJECT` network is based on several individual components.
+The following diagram gives an overview of all components that are needed to run a network and interact with the network.
+
+![Components of a network](../assets/hedera-structure.svg "Components of a network")
+
+The components shown in the diagram will be described in the following sections.
+
+## Consensus Node
+The Consensus Node is the core software that runs a `PROJECT` network like the Hedera public network.
+The node is responsible for maintaining the public ledger, processing requested transactions, holding all accounts and other entities,
+and running the Consensus Service, the Token Service, and the Smart Contract Service to fulfill service requests.
+The node is written in Java and runs on the JVM.
+
+## Mirror Node
+The Mirror Node acts as a query node and stores historical data for the network.
+The node consumes the transaction data provided by Consensus Nodes and may additionally provide value-added services
+such as APIs, auditing, analytics, visibility services, security threat modeling, data monetization services, etc.
+The node can also run additional business logic to support applications.
+
+## Mirror Node Explorer
+A visual Explorer interface built on top of a Mirror node providing a human interface for a network.
+
+## Exchange Rate Tool
+Tool to calculate and send the median of exchange rates from various exchanges to `PROJECT` network to
+establish the cost of transactions.
+
+## JSON-RPC-Relay
+Allows Ethereum clients to interact with a `PROJECT` network.
+This project is an implementation of the Ethereum JSON RPC APIs that delegates to both the
+Consensus and Mirror Nodes to fulfill the JSON RPC Specification.
+
+## Client SDKs (Java, JavaScript, Go, C++, Rust, Swift)
+The SDKs for interacting with a `PROJECT` network: the official distributed consensus platform built using the
+hashgraph consensus algorithm for fast, fair, and secure transactions.
+
+## Solo
+Solo provides a framework and associated tools for conducting end-to-end and full stack tests against a `PROJECT` network.
+
+## Licenses and dependencies of the sub-projects
+All sub-projects of `PROJECT` use the Apache license V2.0 (https://www.apache.org/licenses/LICENSE-2.0.txt).
+Since all the subprojects are part of separate repositories and even created in different programming languages,
+their dependencies differ. Based on that, it doesn’t make sense to provide a global list of dependencies.
+All dependencies and related licenses can be identified by automatic project analysis.
+Today, Snyk is used to create automatic dependency and license overviews of the subprojects.
```

Comment:
Question:
How is the Snyk generated report analyzed or assessed? Is the tool also configured to flag (potentially as part of the CI process) the new dependency introduction that do not follow expected license?

Reply:
I will check and come back with more details.